pkg/pipeline/templates/register.go.tmpl

```diff
@@ -0,0 +1,29 @@
+{{ .Header }}
+
+{{ .GenStatement }}
+
+// Package apis contains Kubernetes API for the provider.
+package apis
+
+import (
+	"k8s.io/apimachinery/pkg/runtime"
+
+	{{ .Imports }}
+)
+
+func init() {
+	// Register the types with the Scheme so the components can map objects to GroupVersionKinds and back
+	AddToSchemes = append(AddToSchemes,
+		{{- range $alias := .Aliases }}
+		{{ $alias }}SchemeBuilder.AddToScheme,
+		{{- end }}
+	)
+}
+
+// AddToSchemes may be used to add all resources defined in the project to a Scheme
+var AddToSchemes runtime.SchemeBuilder
```

Comment:
nit: put declaration of `AddToSchemes` on before `init` func which uses it.

Reply:
Just obeying the existing template here https://github.com/crossplane/provider-template/blob/master/apis/template.go#L27